contracts/HashlistsProtocolCollection.sol

```diff
@@ -61,4 +63,13 @@ contract HashlistsProtocolCollection is LSP8Enumerable {
 
         _mint(curator, bytes32(uint256(uint160(curatedListAddress))), true, "");
     }
+
+    function _beforeTokenTransfer(
```

Comment:
just curious why do you need to do this?

Reply:
LSP8Burnable implementation for this ends up clashing with what's under the hood for enumerable. so we need to be explicit which behavior is expected @Ledespana